ta/arch/arm/link.mk

```diff
@@ -25,7 +25,7 @@ reverse = $(if $(wordlist 2,2,$(1)),$(call reverse,$(wordlist 2,$(words $(1)),$(
 link-ldadd  = $(LDADD)
 link-ldadd += $(addprefix -L,$(libdirs))
 link-ldadd += $(addprefix -l,$(call reverse,$(libnames)))
-ldargs-$(binary).elf := $(link-ldflags) $(objs) $(link-ldadd) $(libgcc$(sm))
```

Comment:
I only make test with STM configuration.
If we have liggcc, we miss "raise" function.
But instead we must use libutee only in TA.

I'll make more tests on QEMU tomorrow.